client/coral-framework/constants/config.js

```diff
@@ -0,0 +1,3 @@
+const configElement = document.querySelector('#data');
+
+export const CONFIG = JSON.parse(configElement ? configElement.textContent : '{}');
```

Comment:
For now just move this logic inside of bootstrap, if we need this config somewhere else, we'll pass it through the context created by the bootstrap service.